src/code/speed_meter.c

```diff
@@ -1,25 +1,60 @@
 #include "global.h"
 #include "vt.h"
 
-volatile OSTime D_8016A520;
-volatile OSTime D_8016A528;
-volatile OSTime D_8016A530;
-volatile OSTime D_8016A538;
-volatile OSTime D_8016A540;
-volatile OSTime D_8016A548;
-volatile OSTime D_8016A550;
-volatile OSTime D_8016A558;
-volatile OSTime gRSPAudioTotalTime;
-volatile OSTime gRSPGFXTotalTime;
-volatile OSTime gRSPOtherTotalTime;
+/**
+ * How much time the audio update on the audio thread (`func_800E4FE0`) took in total, between scheduling the last two
+ * graphics tasks.
+ */
+volatile OSTime gAudioThreadUpdateTimeTotalPerGfxTask;
```

Comment:
Shouldn't this be `gAudioThreadUpdateTimeTotal` to be consistent with the other timers?

Reply:
maybe? but the time period this is measured over is not the same, this is from gfxTask completed to next gfxTask completed (which also means it may accumulate to high values if for example the gfx buffers overlap/overflow and the game skips frames, but that may be an oversight), the others are from `Graph_Update` end to next `Graph_Update` end

Reply:
Hmm I think I understand, it's probably fine as is then, tho I wonder why they did it that way since it kinda defeats the purpose of showing all these timers together if they don't measure things in the same time range